docs(*): add a deprecated message to granular tracing (#7254)
Kong/docs.konghq.com: app/_src/gateway/production/tracing/index.md

In this section, we will describe the tracing capabilities of Kong.

{% if_version gte:3.7.x %}
{:.important}

> > **Important**: The Granular Tracing is deprecated from {{site.base_gateway}} `3.7.0.0` onwards,
> and configurations like `tracing = on` are not available any longer. Instead, use the
> OpenTelemetry Tracing described on this page.

Reviewer: ```suggestion
> **Important**: Granular Tracing is deprecated from {{site.base_gateway}} `3.7.0.0` onward,
and configurations like `tracing = on` are no longer available. Instead, use the
OpenTelemetry Tracing described on this page.
```
